api_benchmark/mono_pkg.yaml

```diff
@@ -6,4 +6,5 @@ stages:
       - format
       - command: ./../tool/setup.sh
       - command: ./compile_protos.sh
+      - analyze:
```

Comment:
What does this empty step do?

Reply:
It analyzes the code, but w/o using `--fatal-infos`.